code/src/pages/products/ProductsPage.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { MemoryRouter } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import ProductsPage from "@/pages/products/ProductsPage";
+import { useGetProductsQuery } from "@/store/api/productsApi";
+
+jest.mock("@/store/api/productsApi", () => ({
+  useGetProductsQuery: jest.fn()
+}));
+
+jest.mock("@/components/product-card/ProductCard", () =>
+  jest.fn(() => <div data-testid="product-card" />)

```

Comment:
> I suggest using AppBox here, what do you think?

It's like a `mock` property and it's not so important here I guess

Reply:
We agreed with our team to use AppBox and you also agreed with it and use it in components code. Please be consistent here, because question with boxes was discussed before

Reply:
> We agreed with our team to use AppBox and you also agreed with it and use it in components code. Please be consistent here, because question with boxes was discussed before

refactored tests